File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        int n = nums.length;
        for(int i =0; i<n; i++) {
            boolean exists = false;

            for(int j=i+1; j<n; j++) {
                if(nums[j] == nums[i]) {
                    exists = true;
                }
            }

            if(exists) return true;
        }

        return false;
    }
}

File: Data Structures & Algorithms/jump-game/submission-1.py
class Solution:
    def canJump(self, nums: List[int]) -> bool:
        if not nums:
            return True
        

        def dfs(i):
            if i >= len(nums) - 1:
                return True
            
            max_jump = nums[i]
            if max_jump == 0:
                return False
            
            for j in range(i+1, i + max_jump + 1):
                if dfs(j):
                    return True
            return False
        
        return dfs(0)
